sci-mathematics/frobby/frobby-0.9.0-r1.ebuild

```diff
@@ -1,55 +0,0 @@
-# Copyright 1999-2013 Gentoo Foundation
-# Distributed under the terms of the GNU General Public License v2
-# $Id$
-
-EAPI=4
-
-inherit eutils
-
-DESCRIPTION="Software system and project for computations with monomial ideals"
-HOMEPAGE="http://www.broune.com/frobby/"
-SRC_URI="http://www.broune.com/frobby/frobby_v${PV}.tar.gz"
-
-LICENSE="GPL-2"
-SLOT="0"
-KEYWORDS="amd64 ~arm x86"
```

Comment:
@Moiman in `sci-*` that's fine, we don't want to make any stable guarantees anymore